### DISCUSSION TITLE
How to Use mpv Command Line to Control Playback, Stop, and Get Progress in a Windows C++ Project？

### DISCUSSION THREAD

**Discussion by fyyhhGit** [zhongfly/mpv-build#119] [Q&A]
I want to use the mpv command line to play video files, control playback progress, and pause, but I am not very familiar with ipc-json.

**Comment by zhongfly** [comment]
I'm not an mpv developer and have never used ipc-json.
You can get help at the mpv repository, as you've already done.
https://github.com/mpv-player/mpv/discussions/15488

**Comment by fyyhhGit** [comment]
Okay, thank you.
